README.md

```diff
@@ -1,27 +1,26 @@
 # template-clarify-automation
 
-This is a template repo for running [Clarify.io](https://clarify.io) automation routines using the [clarify-go](https://github.com/clarify/clarify-go) SDK. It currently include examples for the following routines:
+This is a template repo for running [Clarify.io](https://clarify.io) automation routines using the [clarify-go](https://github.com/clarify/clarify-go) SDK. It currently include examples for the following one routine:
 
-- `publish`: Automatically publish items from signals by filtering which signals to publish and applying transforms.
+- `publish`: Automatically bulk publish items from signals by filtering which signals to publish and apply your own _transforms_ to improve the meta-data.
 
-## Getting started
+## Who should use the publish routine?
 
-This is a _template repository_. To use this code and run it on GitHub, you can start by creating a new (private) repository by clicking "Use this template" button. Next up, you can clone the repository to your local computer and make your own changes.
+If you have been using Clarify for some time, and you are starting to accumulate a lot of [signals](https://docs.clarify.io/developers/quickstart/create-signals), you may need an automated rule-based process for you to publish signals as items. You may want to script how to clean up the raw meta-data from your [integrations](https://docs.clarify.io/users/admin/integrations/) and produce nicely looking [items](https://docs.clarify.io/users/admin/items/) with _normalized_ labels, naming, engineering unit and other properties. Maybe your data sources encode specific information into the name? Maybe they have their own way of encoding engineering units that just doesn't make sense inside Clarify?
 
-If your organization does not use GitHub for hosting code, don't worry. Simply create a clone of this repo on your organizations code management server, or just clone it down locally. Just make sure that you DO NOT push reviews regarding your organization _back_ to the [template repository](https://github.com/clarify/template-clarify-automation/). Pulling down changes as the template repository evolve, is still fine.
+This automation allows you to write a series of _transform functions_ using the Go programming language. Transforms are again organized into rules. Each rule declare a list of integrations to publish from and an optional signal filter. Be sure that the same signal is only handled by one rule! _Transforms_ are functions that apply changes to the raw signal meta-data. We recommend that you write short and simple transforms that _gradually_ improve the meta-data based on previous steps, rather then one large and overly complex transform that does everything in one operation. This also allows you to more easily compose and reuse transforms across multiple rules. However it's your choice!
```

Comment:
```suggestion
This automation allows you to write a series of _transform functions_ using the Go programming language. Transforms are again organized into rules. Each rule declare a list of integrations to publish from and an optional signal filter. Be sure that the same signal is only handled by one rule! _Transforms_ are functions that apply changes to the raw signal meta-data. We recommend that you write short and simple transforms that _gradually_ improve the meta-data based on previous steps, rather than one large and overly complex transform that does everything in one operation. This also allows you to more easily compose and reuse transforms across multiple rules. However, it's your choice!
```